utils/utils.go

```diff
@@ -36,7 +36,7 @@ func RemoveStringFromSlice(a string, slice []string) []string {
 
 // RemoveStringsFromSlice removes all occurrences of strings from slice.
 func RemoveStringsFromSlice(slice []string, strings ...string) []string {
-	newSlice := []string{}
+	var newSlice []string
```

Comment:
`var newSlice []string` and `newSlice := []string{}` are not the same thing if you run it through json.Marshal, one will serialize as `nil` and the other as `[]`. Please revert this and other changes similar to it

Reply:
that's right, it's not the same, but the tests were successful. i reverted all similar changes.